R/derive_vars_duration.R

```diff
@@ -72,11 +72,37 @@
 #'
 #'   Permitted Values: `TRUE`, `FALSE`
 #'
+#' @param type lubridate duration type.
+#'
+#'   See below for details.
+#'
+#'   Default: `"duration"`
```

Comment:
Sorry - I'm really struggling to remember - Should we be purging Defaults from our documentation just so it is one less thing to maintain??  @bundfussr

Reply:
Yes, I think we decided not stating the default value in the parameter description.